{"uuid":"6859696d-45cc-495b-a164-650d6383e787","historyId":"d853e7d3f489162f732172cf3d63ca1c","fullName":"testcases.TestLogin.testLoginWithoutInvalidCredentials","labels":[{"name":"package","value":"testcases.TestLogin"},{"name":"testClass","value":"testcases.TestLogin"},{"name":"testMethod","value":"testLoginWithoutInvalidCredentials"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"C:/Users/ishti/OneDrive/Documents/GitHub/wafilife"},{"name":"subSuite","value":"testcases.TestLogin"},{"name":"host","value":"TANIM"},{"name":"thread","value":"2100@TANIM.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testLoginWithoutInvalidCredentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [https://www.wafilife.com/my-account/] but found [https://www.wafilife.com/my-account]","trace":"java.lang.AssertionError: expected [https://www.wafilife.com/my-account/] but found [https://www.wafilife.com/my-account]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat testcases.TestLogin.testLoginWithoutInvalidCredentials(TestLogin.java:45)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Testing login functionality with invalid credentials","steps":[],"attachments":[{"name":"After trying to logging in with wrong credentials","source":"975d0ae8-2648-4df7-b650-bd6ec218087b-attachment"}],"parameters":[{"name":"arg2","value":"Error: The password field is empty."},{"name":"arg1","value":""},{"name":"arg0","value":"kisivel401@adrais.com"}],"start":1717480014263,"stop":1717480016628}